doc/nestml_language/neurons_in_nestml.rst

```diff
@@ -30,9 +28,8 @@ A neuron model written in NESTML can be configured to receive two distinct types
 .. code-block:: nestml
 
    input:
-     I_stim pA <- continuous
-     AMPA_spikes pA <- spike
-   end
+       I_stim pA <- continuous
+       AMPA_spikes pA <- spike
```

Comment:
Should the unit be removed already in this PR? The same comment also applies further down.

Reply:
I will fix that in a new PR #882